python/tk_multi_workfiles/work_area.py

```
@@ -190,89 +203,117 @@ def resolve_user_sandboxes(self):
-        settings_to_find = ["saveas_default_name", "saveas_prefer_version_up", 
+        settings_to_find = ["saveas_default_name", "saveas_prefer_version_up",
                             "version_compare_ignore_fields", "file_extensions"]
         resolved_settings = {}
-        app = sgtk.platform.current_bundle()
-        try:
-            if self._context:
-                resolved_settings = self._get_settings_for_context(self._context, templates_to_find, settings_to_find)
-        except:
-            app.log_exception("There was an error parsing the settings for context '%s'" % self._context)
-        finally:
-            # update the templates and settings regardless if an exception was raised.
-            #
-
-            # update templates:
-            self.work_area_template = resolved_settings.get("template_work_area")
-            self.work_template = resolved_settings.get("template_work")
-            self.publish_area_template = resolved_settings.get("template_publish_area")
-            self.publish_template = resolved_settings.get("template_publish")
-
-            # update other settings:
-            self.save_as_default_name = resolved_settings.get("saveas_default_name", "")
-            self.save_as_prefer_version_up = resolved_settings.get("saveas_prefer_version_up", False)
-            self.version_compare_ignore_fields = resolved_settings.get("version_compare_ignore_fields", [])
-            extensions = resolved_settings.get("file_extensions") or []
-            extensions = [ext if ext.startswith(".") else ".%s" % ext for ext in extensions if ext]
-            self.valid_file_extensions = extensions
-
-            # test for user sandboxes:
-            self._work_template_contains_user = self.work_template and bool(self._get_template_user_keys(self.work_template))
-            self._publish_template_contains_user = self.publish_template and bool(self._get_template_user_keys(self.publish_template))
+        if self._context:
```

Of all this block, only the if self._context block is interesting. The rest is jus an indentation change. I removed the try catch so exceptions were not swallowed by the except clause and instead reported back to the caller.